examples/calimero-login/config-overrides.js

```diff
@@ -1,10 +1,18 @@
- 
-const webpack = require('webpack'); 
-module.exports = function override(config) { 
-   config.plugins = (config.plugins || []).concat([ 
-   	new webpack.ProvidePlugin({ 
-    	process: 'process/browser', 
-      Buffer: ['buffer', 'Buffer'] 
-    }) 
-   ]) 
-   return config; }
+const webpack = require("webpack")
+
+module.exports = function override(config, env) {
+    config.resolve.fallback = {
+        ...config.resolve.fallback,
+        stream: require.resolve("stream-browserify"),
+        buffer: require.resolve("buffer"),
+    }
+    config.resolve.extensions = [...config.resolve.extensions, ".ts", ".js"]
+    config.plugins = [
+        ...config.plugins,
+        new webpack.ProvidePlugin({
```

Comment:
https://stackoverflow.com/questions/68707553/uncaught-referenceerror-buffer-is-not-defined